cookbooks/continental_extension.prm

```diff
@@ -0,0 +1,254 @@
+#   Geological Society of London Special Publications, v.24, p.63-70.
+# The initial constraints are:
+#   Layer Surface Temperature - upper crust (ts1) = 273 K; 
+#                               mantle      (ts3) = 823 K;  
+#   Model Base Temperature - (tb) = 1573 K;
+#   Heat Production - upper crust (A) = 1.5e-6 W/m^3; 
+#   Thermal Conductivity - upper crust (k1) = 2.5 (W/(m K)); 
+#                          lower crust (k2) = 2.5 (W/(m K)); 
+#                          mantle      (k3) = 3.3 (W/(m K));
+# To satisfy these constraints, the following values are required:
+#   Layer Surface Heat Flow - upper crust (qs1) = 0.065357 W/m^2; 
+#                             lower crust (qs2) = 0.035357 W/m^2; 
+#                             mantle      (qs3) = 0.035357 W/m^2;
+#   Temperature - base of upper crust (ts2) = 681.5714
+subsection Initial conditions
+  set Model name = function
+  subsection Function
+    set Variable names = x,y
+    set Function constants = h=100e3,ts1=273,ts2=681.5714,ts3=823., \
+                                     k1=2.5,k2=2.5,k3=3.3,A=1.5e-6, \
+                             qs1=0.0653571,qs2=0.035357,qs3=0.035357,qb3=0.035357
+    set Function expression = if( (h-y)<=20.e3, \
+                                  ts1 + (qs1/k1)*(h-y) - (A*(h-y)*(h-y))/(2.0*k1), \
+                                  if((h-y)>20.e3 && (h-y)<=30.e3, ts2 + (qs2/k2)*(h-y-20.e3),\
+                                  ts3 + (qs3/k3)*(h-y-30.e3)));
+  end
+end
+
+# Internal heating (only internal heating for crust)
+# Assuming a very large decay time, the heat production (W/m^3) is effectively:
+#   radioactive_heating_rate (W/kg) * density (kg/m^3)
+# For a reference crustal density of 2800 kg/m^3 and 1.5e-6 W/m^3 heating production,
+# the heating rate (assuming a single element) is 1.5e-6/2800. = 5.357142857e-10 ~= 5.357e-10
+subsection Heating model
+  set Model name = radioactive decay
+  subsection Radioactive decay
+    set Number of elements    = 1
+    set Heating rates                 = 5.357e-10
+    set Half decay times              = 1.e20
+    set Initial concentrations mantle = 0.0
+    set Initial concentrations crust  = 1.0
+    set Crust defined by composition  = true
+    set Crust composition number      = 0
+  end
+end
+
+# Material model
+# Rheology: Non-linear viscous flow and Drucker Prager Plasticity
+# Values for most rheological parameters are specified for a background material and
+# each compositional field.  Values for viscous deformation are based on dislocation
+# creep flow-laws, with distinct values for the upper crust (wet quartzite), lower
+# crust (wet anorthite) and mantle (dry olivine).  Table 1 of Naliboff and Buiter (2015),
+# Earth Planet. Sci. Lett., v.421, p. 58-67 contains values for each of these flow laws.     
+subsection Material model
+  set Model name = visco plastic
+
+  subsection Visco Plastic
+
+    # Reference temperature and viscosity
+    set Reference temperature = 293
```

Comment:
Good eye again!  The `Reference temperature` here is indeed referring to the densities at 293 K.  In lithospheric scale models with the Buossinesq approximation, people typically use a value of 273 K (0 Celsius) for crustal layers and in some cases mantle and asthenospheric layers.  In some studies, people will use an intermediate value (ex: 600 Celcius) for the upper mantle layers.  In this case, 293 K was the default in a range of material models in ASPECT and I adopted it, although in the past I've used 273 K.  273 K may be a bit more reasonable for Colorado right now :)